### PR TITLE
Keyrings

### PR DESCRIPTION
**Pull request by MDarmawan**
Add force param

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# IBM Cloud Go SDK Version 0.10.0
+# IBM Cloud Go SDK Version
```
Comment (BrunoHenriques) [inline README.md:1]: Did you mean just  "IBM Cloud Go SDK"?

File: key_rings.go [hunk 2]
```diff
@@ -57,11 +59,24 @@ func (c *Client) GetKeyRings(ctx context.Context) (*KeyRings, error) {
 	return &rings, nil
 }
 
+type DeleteKeyRingQueryOption func(*http.Request)
+
+func UseForce(force bool) DeleteKeyRingQueryOption {
```
Comment (BrunoHenriques) [inline key_rings.go:64]: Should the name of the function be `UseForce` or `WithForce`? The With prefix seems to be the standard or at least what is in use in other similar functions.

**Review by BrunoHenriques — APPROVED** [review]

**Review by stephaniegalang — APPROVED** [review]
Love the extension of options using variadic args to get around backwards compatibility 🌟

**Review by william8siew — APPROVED** [review]
Looks good. I did notice
... for DeleteKeyRingQueryOption. 
It implies we might have more options in the future 👀 , i like the future thinking

**Pull request merged**